be/src/exec/vectorized/hdfs_scan_node.cpp

```diff
@@ -54,7 +65,7 @@ Status HdfsScanNode::init(const TPlanNode& tnode, RuntimeState* state) {
 
     const auto& slots = _tuple_desc->slots();
     for (size_t i = 0; i < slots.size(); i++) {
-        if (_hdfs_table->is_partition_col(slots[i])) {
+        if (_hdfs_table != NULL && _hdfs_table->is_partition_col(slots[i])) {
```

Comment:
```suggestion
        if (_hdfs_table != nullptr && _hdfs_table->is_partition_col(slots[i])) {
```
Prefer nullptr to NULL after c++11